101-vm-with-standardssd-disk/azuredeploy.json

```diff
@@ -0,0 +1,249 @@
+  },
+  "resources": [
+      
+      {
+          "apiVersion": "[variables('apiVersion')]",
+          "type": "Microsoft.Network/publicIPAddresses",
+          "sku": {
+              "name": "Basic",
+              "tier": "Regional"
+          },
+          "name": "[variables('publicIpAddressName')]",
+          "location": "[parameters('location')]",
+          "properties": {
+              "publicIPAddressVersion": "IPv4",
+              "publicIPAllocationMethod": "[variables('publicIPAddressType')]"}
+          
+      },
+      {
+          "apiVersion": "[variables('apiVersion')]",
+          "type": "Microsoft.Network/virtualNetworks",
+          "name": "[parameters('virtualNetworkName')]",
+          "location": "[parameters('location')]",
+          "properties": {
+              "addressSpace": {
+                  "addressPrefixes": [
+                      "[variables('addressPrefix')]"
+                  ]
+              },
+              "subnets": [
+                  {
+                      "name": "[parameters('subnetName')]",
+                      "properties": {
+                          "addressPrefix": "[variables('subnetPrefix')]"
+                      }
+                  }
+              ]
+          }
+      },
+      {
+          "apiVersion": "[variables('apiVersion')]",
+          "type": "Microsoft.Network/networkInterfaces",
+          "name": "[variables('networkInterfaceName')]",
+          "location": "[parameters('location')]",
+          "dependsOn": [
+              "[concat('Microsoft.Network/publicIPAddresses/', variables('publicIpAddressName'))]",
+              "[concat('Microsoft.Network/virtualNetworks/', parameters('virtualNetworkName'))]"
+          ],
+          "properties": {
+              "ipConfigurations": [
+                  {
+                      "name": "ipconfig1",
+                      "properties": {
+                          "privateIPAllocationMethod": "Dynamic",
+                          "publicIPAddress": {
+                              "id": "[resourceId('Microsoft.Network/publicIPAddresses',variables('publicIpAddressName'))]"
+                          },
+                          "subnet": {
+                              "id": "[variables('subnetRef')]"
+                          }
+                      }
+                  }
+              ]
+          }
+      },
+      {
+          "apiVersion": "[variables('computeResouresApiVersion')]",
+          "type": "Microsoft.Compute/virtualMachines",
+          "name": "[parameters('virtualMachineName')]",
+          "location": "[parameters('location')]",
+          "dependsOn": [
+              "[concat('Microsoft.Network/networkInterfaces/', variables('networkInterfaceName'))]"
+          ],
+          "properties": {
+              "hardwareProfile": {
+                  "vmSize": "[parameters('virtualMachineSize')]"
+              },
+              "osProfile": {
+
```

Comment:
format the JSON (e.g. SHIFT+ALT+F in VS Code)

Reply:
Done